### ISSUE TITLE
`gh-branch` makes repository too large

### ISSUE THREAD

**Issue by epassaro** [tardis-sn/tardis#1579]
I noticed `tardis` repository is ~400MB. The reason is because `gh-branch` is too large.

**`master` branch size:** 64.30MiB
```
git clone -b master --single-branch https://github.com/tardis-sn/tardis.git master

Cloning into 'master'...
remote: Enumerating objects: 22043, done.
remote: Counting objects: 100% (76/76), done.
remote: Compressing objects: 100% (58/58), done.
remote: Total 22043 (delta 29), reused 44 (delta 16), pack-reused 21967
Receiving objects: 100% (22043/22043), 64.30 MiB | 2.95 MiB/s, done.
Resolving deltas: 100% (15780/15780), done.

```

**`gh-pages` branch size:** 312.82MiB

```
git clone -b gh-pages --single-branch https://github.com/tardis-sn/tardis.git

Cloning into 'tardis'...
remote: Enumerating objects: 52088, done.
remote: Counting objects: 100% (3067/3067), done.
remote: Compressing objects: 100% (1040/1040), done.
remote: Total 52088 (delta 2414), reused 2407 (delta 1909), pack-reused 49021
Receiving objects: 100% (52088/52088), 312.82 MiB | 3.14 MiB/s, done.
Resolving deltas: 100% (41922/41922), done.
Updating files: 100% (749/749), done.
```

I propose use `force-orphan` parameter available in the Action `peaceiris/actions-gh-pages`. Then the `gh-pages` always contains just a single commit.

**Comment by andrewfullard** [comment]
@epassaro is this fixed now?

**Comment by epassaro** [comment]
nope, I just proposed a solution. I think there's nothing wrong with making `gh-pages` orphan because GitHub Pages just render the latest commit.

But I wanted to hear some opinions before doing it.

**Comment by wkerzendorf** [comment]
I think that sounds sensible

**Comment by epassaro** [comment]
Closed by #1651
